lib/linebuffer.js

```diff
@@ -167,7 +168,8 @@ module.exports.setSocket = function(sock) {
         exports.resetStats();
         exclude_regex = config.exclude_regex && new RegExp(config.exclude_regex);
 
-        if (config.HTTP_RETRY && os.platform() !== 'win32') {
+        if (config.HTTP_RETRY) {
+            if (!fs.existsSync(config.FILEQUEUE_PATH)) fs.mkdirSync(config.FILEQUEUE_PATH);
```

Comment:
This line cannot be here. Linebuffer loop runs for every log line and this would kill performance.